documentation/service_admin/assembly-plans.adoc

```diff
@@ -0,0 +1,25 @@
+// This assembly is included in the following assemblies:
+//
+// <List assemblies here, each on a new line>
+:parent-context: {context}
+
+include::common/attributes.adoc[]
```

Comment:
Don't include this anywhere but master.adoc because of the issues we previously discussed.

Reply:
done